storage/go110_test.go

```diff
@@ -45,6 +46,8 @@ func TestInvoke(t *testing.T) {
 		{2, &googleapi.Error{Code: 599}, &googleapi.Error{Code: 428}},
 		{1, &url.Error{Op: "blah", URL: "blah", Err: errors.New("connection refused")}, nil},
 		{1, io.ErrUnexpectedEOF, nil},
+		{1, fmt.Errorf("Test unwrapping of a temporary error: %w", &googleapi.Error{Code: 500}), nil},
```

Comment:
This is the correct way to test but it's not available in Go < 1.13, and we need to support back to 1.11. I believe you can use https://pkg.go.dev/golang.org/x/xerrors as a workaround.